lib/ProductOpener/Display.pm

```diff
@@ -6923,8 +6923,28 @@ sub display_field($$) {
 			$value =~ s/\n/<br>/g;
 		}
 	}
+	
+	my $to_do_status = '';
+	my $done_status = '';
+
+	if ($field eq 'states'){
+		my @to_do_status;
+		my @done_status;
+		my $state_items = $product_ref->{$field . "_hierarchy"};
+		foreach my $val (@$state_items){
+			if ((index($val, "to-") != -1) or (index($val, "empty") != -1)) {
+				push(@to_do_status, $val);
+			}
+			else {
+					push(@done_status, $val);
+			}
+			
+		}
+		$to_do_status = display_tags_hierarchy_taxonomy($lc, $field, \@to_do_status);
+		$done_status = display_tags_hierarchy_taxonomy($lc, $field, \@done_status);
+	}
 
-	if (defined $taxonomy_fields{$field}) {
+	if (defined $taxonomy_fields{$field} and $field ne 'states') {
```

Comment:
Can we use an `elif`?

Reply:
Yes, would be better, will do that. Thanks